includes/features/onboarding/class-wc-admin-onboarding.php

```diff
@@ -215,14 +212,22 @@ public static function add_uploaded_theme_data( $response ) {
 	 * @return bool
 	 */
 	public static function has_woocommerce_support( $theme ) {
-		$directory = new RecursiveDirectoryIterator( $theme->theme_root . '/' . $theme->stylesheet );
-		$iterator  = new RecursiveIteratorIterator( $directory );
-		$files     = new RegexIterator( $iterator, '/^.+\.php$/i', RecursiveRegexIterator::GET_MATCH );
-
-		foreach ( $files as $file ) {
-			$content = file_get_contents( $file[0] );
-			if ( preg_match( '/add_theme_support\(([^(]*)(\'|\")woocommerce(\'|\")([^(]*)/si', $content, $matches ) ) {
-				return true;
+		$themes = array( $theme );
```

Comment:
Wow I'm really surprised there isn't a WP core util for checking theme support on non-active themes 🤷‍♂ - but in my googlin' and checking https://core.trac.wordpress.org/browser/tags/5.2.1/src/wp-includes/theme.php I don't see anything outside of `current_theme_supports()`

Reply:
Ya, I thought for sure there would be one too.  I guess since support is declared in PHP instead of template meta tags, it would require loading an entire theme which may cause side effects.